Level_2/string_compression.py

```diff
@@ -0,0 +1,26 @@
+def solution(s: str) -> int:
+    if len(s) == 1:
+        return 1
+    answer = []
+    for i in range(1, (len(s)//2) + 1):
+        temp = s[:i]
+        cnt = 1
+        st = ""
+        for j in range(i, len(s), i):
+            if temp == s[j:i+j]:
+                cnt += 1
+            else:
+                if cnt != 1:
+                    st += str(cnt) + temp
+                    temp = s[j:i+j]
+                    cnt = 1
+                else:
+                    st += temp
+                    temp = s[j:i+j]
+                    cnt = 1
```

Comment:
중복되는 코드! 
21~24번 줄도 한줄로 작성 가능! 

```suggestion
                temp = s[j:i+j]
                cnt = 1  
                st += (str(cnt) + temp if cnt > 1 else  temp)
```